packages/grid/src/vaadin-grid.js

```diff
@@ -134,9 +134,8 @@ import { StylingMixin } from './vaadin-grid-styling-mixin.js';
  * The `<vaadin-grid>` calls this function lazily, only when it needs more data
  * to be displayed.
  *
- * See the [`dataProvider`](#/elements/vaadin-grid#property-dataProvider) in
- * the API reference below for the detailed data provider arguments description,
- * and the “Assigning Data” page in the demos.
```

Comment:
There are no more demo pages, so the last part of this sentence was obsolete - rephrased it.